lib/assessment.rb

```diff
@@ -5,7 +5,7 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.reduce { |a, b| a.length <=> b.length }
```

Comment:
Close. With this kind of block, you don't want to use `reduce`, but rather an alternative method. However, it is possible to solve this one with `reduce`.